README

```diff
@@ -1,3 +1,15 @@
+Docker Project
+
+A dockerfile to preinstall "vim" and "build-essential".
+
+Usage:
+1) cd docker_project
+2) docker build -t installation:1.0 .
+3) docker run -it installation:1.0 /bin/bash
+
+
+Scripts
+
 There are 4 scripts in this project. 
```

Comment:
After moving scripts to the directory,  update this line, so that user could understand where are located that 4 scripts